components/library/tntnet/Makefile

```diff
@@ -37,6 +37,11 @@ include $(WS_TOP)/make-rules/ips.mk
 COMPONENT_PRE_CONFIGURE_ACTION = \
         ($(CLONEY) $(SOURCE_DIR) $(@D))
 
+COMPONENT_POST_INSTALL_ACTION += \
+        (cd $(PROTO_DIR)/etc/tntnet && \
+         $(GSED) -i 's|>www-data<|>webservd<|g' tntnet.xml && \
+         $(GSED) -i 's|/$(MACH64)/|/|g' tntnet.xml )
+
```

Comment:
I'd rather not overcomplicate. Delivering an extra custom copy of config is
extra maintenance load for later updates when new keywords are added etc.

This default config and sample webapp are moderately useless - just an
illustrative aid to get a developer started, to make his own webapp with
its servlets, ssl keys, logdirs and other bits of config...

--
Typos courtesy of GMail on my Samsung Android
24.03.2016 16:12 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/library/tntnet/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/1777#discussion_r57331886>
> :
>
> > @@ -37,6 +37,11 @@ include $(WS_TOP)/make-rules/ips.mk
> >  COMPONENT_PRE_CONFIGURE_ACTION = \
> >          ($(CLONEY) $(SOURCE_DIR) $(@D))
> >
> > +COMPONENT_POST_INSTALL_ACTION += \
> > +        (cd $(PROTO_DIR)/etc/tntnet && \
> > +         $(GSED) -i 's|>www-data<|>webservd<|g' tntnet.xml && \
> > +         $(GSED) -i 's|/$(MACH64)/|/|g' tntnet.xml )
> > +
>
> perhaps, better to patch tntnet.xml.in or deliver config as separate file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1777/files/cc739bc398ec10c740b0d3937f6723360fc64c71..feab6b05a8cb6be9b01bbebadf5672a13c336885#r57331886>
>